home-work.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"unicode/utf8"
+)
+
+func average(arr []int) float64 { // Task 1
+	sum := 0
+	for _, a := range arr {
+		sum += a
+	}
+	return float64(sum) / float64(len(arr))
+}
+
+func max(words []string) string { // Task 2.1
+	word := ""
+	for _, a := range words {
+		if utf8.RuneCountInString(a) > utf8.RuneCountInString(word) {
+			word = a
+		}
+	}
+	return word
+}
+
+func reverse(arr []int64) []int64 { // Task 2.2
+	ar := append(arr, 0)
```

Comment:
Too complicated implementation.
Please, create new slice explicitly (with same length and capacity as the input slice, and fill it out.